Makefile

```diff
@@ -509,7 +509,7 @@ ifeq ($(cc-name),clang)
 ifneq ($(CROSS_COMPILE),)
 CLANG_TRIPLE    ?= $(CROSS_COMPILE)
 CLANG_FLAGS	:= --target=$(notdir $(CLANG_TRIPLE:%-=%))
-GCC_TOOLCHAIN_DIR := $(dir $(shell which $(LD)))
+GCC_TOOLCHAIN_DIR := $(dir $(shell which $(CROSS_COMPILE)elfedit))
```

Comment:
I've changed it to `readelf` to prevent regressions with the `Android.mk` file, unless @jerpelea wants to add [the elfedit symlink commits](https://android-review.googlesource.com/q/topic:%22elfedit-symlink%22) that you guys apply, in which I can change it back to actually match upstream.